zcash_client_sqlite/src/wallet/orchard.rs

```diff
@@ -399,6 +399,11 @@ pub(crate) mod tests {
         testing::pool::send_single_step_proposed_transfer::<OrchardPoolTester>()
     }
 
+    #[test]
+    fn send_with_multiple_change_outputs() {
+        testing::pool::send_with_multiple_change_outputs::<SaplingPoolTester>()
```

Comment:
```suggestion
        testing::pool::send_with_multiple_change_outputs::<OrchardPoolTester>()
```

Reply:
I spotted a similar bug in an existing test while rebasing another PR; we may want to check them all.